data/transport missions.txt

```diff
@@ -3404,3 +3404,69 @@ mission "Capture Smuggler"
+		payment -200000
+	on offer
+		conversation
+			`You notice a computer terminal blinking in the corner of an alley.`
+			choice
+				`	(Check it out.)`
+				`	(Ignore it.)`
+					decline
+			`When you approach, you are greeted by a voice from the terminal; "Dear Sir, Madam, please respond!`
+			choice
+				`	"Hello"`
+				`	(Ignore it and walk away.)`
+					decline
+			`	"My name is Mr. Smith and I'm contacting you with a highly confidential and urgent proposal. I am, or more accurately was, the financial manager of the recently deceased warlord Limping Pappa. I am looking for a highly trustworthy individual who can help me relocate Mr. Pappa's fortune."`
+			`	"Mr. Pappa acquired over 300 million credits."`
+			choice
+				`	"How did Mr. Pappa acquire his fortune exactly?"`
+				`	"What are you proposing?"`
+					goto proposal
+				`	"This sounds like a credits-stealing scam, I will not fall for this"`
+					decline
+			`	"Oh, euuh, Mr. Pappa was specialized into tricking rich victims to transfer credits to him.`
+			label proposal
+			`	"I obviously cannot make myself a heir to Mr. Pappa. So what I propose is to present you as heir and have the inheritance transferred to you.`
+			`	"When you receive the inheritance on <destination>, then you send half of it back to me."`
+			choice
+				`	"Okay, let's do this."`
+				`	"Sounds too good to be true, what is the catch?"`
+				`	"Sorry, this sounds too illegal for me."`
+					decline
+			`	Mr. Smith responds: "There is one small problem with the inheritance tax, it needs to be paid upfront. Can you pay 200,000 credits so that I can get the process going?"`
+			choice
+				`	"Sure, no problem."`
+					goto pay_upfront
+				`	"Are you trying to trick me?"`
+				`	"Sounds too risky, I don't want to pay anything up-front."`
+					goto no_deal
+			`	"Of course I don't try to trick you! Mr. Pappa's victims were all persons that had more credits than common sense. But you look a lot smarter than any of Mr. Pappa's victims.`
+			`	"Think of the huge amount of credits you will receive for just acting as heir! Can I count on you to pay the tax and make both of us very rich?"`
+			choice
+				`	"Sure, no problem."`
+					goto pay_upfront
+				`	"Sounds too risky, I don't want to pay anything up-front."`
+					goto no_deal
+			label pay_upfront
+			`	The terminal suddenly goes black after you transfer 200,000 credits to the account given by Mr. Smith.`
+			`	You wonder if you did the right thing.`
+				accept
+			label no_deal
+			`	The terminal suddenly goes black. Mr. Smith is apparently no longer interested in doing business with you.`
+				decline
+	on complete
+		dialog "There is no inheritance for Mr. Pappa waiting for you at the bank, but there are many other victims that paid some credits while expecting to earn a larger amount of credits in return. Some other victims make a big fuss and the alerted authorities fine those victims for attempted inheritance fraud. You leave the bank in silence."
```

Comment:
> Some other victims make a big fuss and the alerted authorities fine those victims for attempted inheritance fraud.

Somehow this sentence feels clunky to me. I'm not sure if it's due to the third use of the word victim, or the fact that the authorities seem to be there at a moment's notice. Some ideas to iterate on:

> Some other victims make a big fuss and are in turn fined for attempted inheritance fraud by the authorities.

> There is no inheritance for Mr. Pappa waiting for you at the bank, but there are many other victims that paid some credits while expecting to earn a larger amount of credits in return. Some make a big fuss about it, until the alerted authorities start fining them for attempted inheritance fraud. You leave the bank in silence.

> There is no inheritance for Mr. Pappa waiting for you at the bank, but there are many other victims that paid some credits while expecting to earn a larger amount of credits in return. Some local militias are standing at the counter, apparently to keep the croud in check. You see some victims make a big fuss, until the militias fine them for attempted inheritance fraud. You leave the bank in silence.

Reply:
I like the second version you listed and used it as new text. The mission can end in republic space and local militia would be a bit odd there.